ocf_datapipes/load/nwp/nwp.py

```diff
@@ -26,15 +26,85 @@ def __init__(
+            "convective_precipitation": (0, 1000),  # mm/day
+            "snowfall": (0, 1000),  # mm water equivalent/day
+            "graupel": (0, 500),  # mm water equivalent/day
+            "cloud_cover": (0, 100),  # Percentage
+            "surface_temperature": (-90, 60),  # Celsius
+            "sea_surface_temperature": (-2, 35),  # Celsius
+            "soil_temperature": (-50, 60),  # Celsius
+            "soil_moisture": (0, 1),  # m^3/m^3
+            "visibility": (0, 100000),  # meters
+            "wind_gust": (0, 250),  # m/s
+            "solar_radiation": (0, 1500),  # W/m^2
+            "longwave_radiation": (0, 750),  # W/m^2
+            "evaporation": (0, 50),  # mm/day
+            "potential_evaporation": (0, 100),  # mm/day
+            "boundary_layer_height": (0, 5000),  # meters
+            "cape": (0, 10000),  # J/kg
+            "cin": (0, 1000),  # J/kg
+            "lifted_index": (-15, 15),  # Kelvin
+            "total_column_water": (0, 100),  # kg/m^2
+            "ozone_concentration": (0, 1000),  # Dobson units
+            "dew_point_temperature": (-100, 35),  # Celsius
+            "wet_bulb_temperature": (-100, 35),  # Celsius
+            "potential_temperature": (0, 1000),  # Kelvin
+            "equivalent_potential_temperature": (0, 1000),  # Kelvin
+            "vorticity": (-1e-3, 1e-3),  # 1/s
+            "divergence": (-1e-3, 1e-3),  # 1/s
+            "vertical_velocity": (-50, 50),  # m/s
+            "cloud_base_height": (0, 20000),  # meters
+            "cloud_top_height": (0, 20000),  # meters
+            "cloud_water_content": (0, 5),  # g/kg
+            "ice_water_content": (0, 5),  # g/kg
+            "surface_roughness": (0, 10),  # meters
+            "albedo": (0, 1),  # dimensionless
+            "friction_velocity": (0, 5),  # m/s
+            "sensible_heat_flux": (-500, 500),  # W/m^2
+            "latent_heat_flux": (-500, 500),  # W/m^2
+            "momentum_flux": (-10, 10),  # N/m^2
+            "surface_pressure": (300, 1100),  # hPa
+            "mean_sea_level_pressure": (870, 1090),  # hPa
+            "tropopause_pressure": (50, 500),  # hPa
+            "tropopause_temperature": (-100, 0),  # Celsius
+            "precipitable_water": (0, 100),  # mm
+            "total_cloud_cover": (0, 100),  # Percentage
+            "low_cloud_cover": (0, 100),  # Percentage
+            "medium_cloud_cover": (0, 100),  # Percentage
+            "high_cloud_cover": (0, 100),  # Percentage
+            "convective_available_potential_energy": (0, 10000),  # J/kg
+            "convective_inhibition": (0, 1000),  # J/kg
+            "storm_relative_helicity": (-1000, 1000),  # m^2/s^2
+            "bulk_richardson_number": (-10, 10),  # dimensionless
+            "lifted_condensation_level": (0, 5000),  # meters
+            "level_of_free_convection": (0, 20000),  # meters
+            "equilibrium_level": (0, 20000),  # meters
+            "UKV": (250, 330),  # UKV specific
```

Comment:
I would remove UKV, im not sure what this is

Reply:
I think im happy with these. Just one question, where did the variable names come from? 

I wonder if we should try to cover all the variables in https://github.com/openclimatefix/ocf_datapipes/tree/main/tests/data

We can always add more in a different PR if you prefer to add the other variables in the variables

Reply:
Ah, super, 

For plascing the limts, its in the channel names. 't' here is short for temperature. The other variables we use are saved [here](https://huggingface.co/openclimatefix/pvnet_uk_region/blob/main/data_config.yaml#L27) and [here](https://huggingface.co/openclimatefix/pvnet_uk_region/blob/main/data_config.yaml#L53). There might be a few others, but we could atleast cover these ones first

Reply:
unfortunately in the test data, we didnt keep all the variables, because we wanted to keep the data small

Reply:
hmm, thats an interesting one. I lik the idea of keeping the repo not dependent on huggingface, but then it is a good palce to store data